packages/SwingSet/src/controller/controller.js

```diff
@@ -258,21 +265,11 @@ export async function makeSwingsetController(
    * @param {BundleID} [allegedBundleID]
    * @returns {Promise<BundleID>}
    */
-  async function validateAndInstallBundle(bundle, allegedBundleID) {
+  async function validateAndInstallBundle(bundle, allegedBundleID = undefined) {
     // TODO The following assertion may be removed when checkBundle subsumes
     // the responsibility to verify the permanence of a bundle's properties.
     // https://github.com/endojs/endo/issues/1106
-
-    Object.values(Object.getOwnPropertyDescriptors(bundle)).every(
-      ({ value, get, writable, configurable }) =>
-        typeof value === 'string' &&
-        get === undefined &&
-        // @ts-ignore `isFake` purposely omitted from type
-        (harden.isFake || (writable === false && configurable === false)),
-    ) ||
-      Fail`Bundle with alleged ID ${q(
-        allegedBundleID,
-      )} must be a frozen object with only string value properties, no accessors`;
+    mustMatch(endoZipBase64Sha512Pattern, bundle);
```

Comment:
The signature of `mustMatch` is (specimen, pattern).
```suggestion
    mustMatch(bundle, endoZipBase64Sha512Pattern);
```